apps/condo/domains/ticket/schema/GetAllResidentTicketsService.js

```diff
@@ -6,10 +6,12 @@ const { GQLCustomSchema } = require('@core/keystone/schema')
 const access = require('@condo/domains/ticket/access/GetAllResidentTicketsService')
 const { generateQuerySortBy } = require('@condo/domains/common/utils/codegeneration/generate.gql')
 const { generateQueryWhereInput } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+const { get } = require('lodash')
 
 const fieldsObj = {
     id: 'ID',
     number: 'Int',
+    property: 'Property',
     status: 'TicketStatus',
     source: 'TicketSource',
     classifier: 'TicketClassifier',
```

Comment:
We have TicketPlaceClassifier and TicketCategoryClassifier 
classifier is deprecated

Reply:
ok, updated input and output ResidentTicket with new classifiers types in 84598b9